core/templates/dev/head/components/rule_editor_directive.html

```diff
@@ -28,8 +28,10 @@
     </div>
 
     <div ng-if="rule.type == 'FuzzyMatches'">
-      <classifier-panel rule-inputs="rule.inputs" on-training-data-deletion="onDeleteTrainingDataEntry(index)">
```

Comment:
Just to follow up on this -- I'm going to punt on this review till you and @anmolshkl have had a chance to talk with @BenHenning. (It would probably be a good idea to have a clear idea for the UI in mind before the meeting, so that you know what questions to ask / what points to bring up.)